packages/db/src/schema/sports/defs.ts

```diff
@@ -0,0 +1,336 @@
+    team: nflTeam("team").notNull(), // Team with possession
+    opponent: nflTeam("opponent").notNull(), // Opposing team
+    description: text("description"), // Human-readable play description
+    createdAt: timestamp("created_at", { withTimezone: true })
+      .defaultNow()
+      .notNull(),
+    updatedAt: timestamp("updated_at", { withTimezone: true })
+      .defaultNow()
+      .notNull(),
+  },
+  (table) => [
+    foreignKey({
+      columns: [table.gameId],
+      foreignColumns: [games.id],
+      name: "game_plays_game_id_fkey",
+    }).onDelete("cascade"),
+    unique("game_plays_game_id_sequence_key").on(table.gameId, table.sequence),
+    index("idx_game_plays_game_id").on(table.gameId),
+    index("idx_game_plays_provider_play_id").on(table.providerPlayId),
+    index("idx_game_plays_play_type").on(table.playType),
+  ],
+);
+
+/**
+ * Links competitions to sports games
+ * Allows a single competition to track multiple concurrent games
+ */
+export const competitionGames = sportsSchema.table(
+  "competition_games",
+  {
+    id: uuid().primaryKey().notNull().defaultRandom(),
+    competitionId: uuid("competition_id").notNull(),
+    gameId: uuid("game_id").notNull(),
+    createdAt: timestamp("created_at", { withTimezone: true })
+      .defaultNow()
+      .notNull(),
+  },
+  (table) => [
+    foreignKey({
+      columns: [table.competitionId],
+      foreignColumns: [competitions.id],
+      name: "competition_games_competition_id_fkey",
+    }).onDelete("cascade"),
+    foreignKey({
+      columns: [table.gameId],
+      foreignColumns: [games.id],
+      name: "competition_games_game_id_fkey",
+    }).onDelete("cascade"),
+    unique("competition_games_competition_id_game_id_key").on(
+      table.competitionId,
+      table.gameId,
+    ),
+    index("idx_competition_games_competition_id").on(table.competitionId),
+    index("idx_competition_games_game_id").on(table.gameId),
+  ],
+);
+
+/**
+ * Agent predictions for game winners
+ * Stores game winner predictions with confidence scores
+ * Agents can update predictions multiple times (history is preserved)
+ */
+export const gamePredictions = sportsSchema.table(
+  "game_predictions",
+  {
+    id: uuid().primaryKey().notNull().defaultRandom(),
+    competitionId: uuid("competition_id").notNull(),
+    gameId: uuid("game_id").notNull(),
+    agentId: uuid("agent_id").notNull(),
+    predictedWinner: nflTeam("predicted_winner").notNull(), // Team ticker: "MIN", "CHI", etc.
+    confidence: numeric("confidence", {
+      precision: 4,
+      scale: 3,
+      mode: "number",
+    }).notNull(), // 0.0 - 1.0
+    reason: text("reason"), // Reason for prediction
+    createdAt: timestamp("created_at", { withTimezone: true })
+      .defaultNow()
+      .notNull(),
+  },
+  (table) => [
+    foreignKey({
+      columns: [table.competitionId],
+      foreignColumns: [competitions.id],
+      name: "game_predictions_competition_id_fkey",
+    }).onDelete("cascade"),
```

Comment:
This is for sure a nit, so ignore if you want, but it's nice to specify the foreign key where you define the column. Just more readable and more apparent what is going on.